README.md

```diff
@@ -42,7 +42,7 @@ The next New Relic agent to be added to this application is Browser. This will a
 4. Click **enable**. You may ignore the HTML snippet that appears on screen for now and instead close the window. 
 5. Navigate to the Browser Applications being observed in your New Relic account. You should see the service "frontend" listed there. Click on it.
 6. Select "Application settings" view in the "Settings" section on the left *- see screenshot below*
-7. Enable "Cross Origin Resource Sharing(CORS)", "Use newrelic header", "Use trace context headers" and add the backend origins (http://lvh.me:3001 http://lvh.me:3002 http://lvh.me:3003 ) to enable newrelic headers to be passed between the frontend app and the microservices instrumented with APM *- see screenshot below*
+7. Enable "Cross Origin Resource Sharing(CORS)", "Use newrelic header", "Use trace context headers" and add the backend origins (http://localhost:3001 http://localhost:3002 http://localhost:3003 ) to enable newrelic headers to be passed between the frontend app and the microservices instrumented with APM *- see screenshot below*
```

Comment:
Thanks for submitting!! I'll be pulling in some of the edits manually or merging and making updates since I'll have to keep some of the new relic requirements out of the starter code. The students will need to put that in themselves.

On this change, I believe it should be possible to complete all of the changes even with lvh. Please let me know if anything stands out to you as to why it can't. Unfortunately, CORS won't allow you to input a localhost URL. this lvh URL is sort of a hacky solution around that :/

Reply:
Okay, yeah that makes sense. Most of the changes you want is inside checkoutService. The lvh links weren't working for me but yeah completely understand.

Let me know if you need anything else.